Modules/trigger.py

```diff
@@ -83,3 +100,24 @@ def channel(self) -> str or None:
             msg (str): Message to send.
         """
         await self.bot.message(self.channel if self.channel else self.nickname, msg)
+
+    def __eq__(self, other) -> bool:
+        if self is other:
+            return True
+        elif isinstance(other, Trigger):
+            for name, value in Trigger.__dict__.items():
```

Comment:
Why are we calling a magic rather than comparing the relevant attributes by name?

Reply:
Short answer: I'm lazy and this object has a bunch of attributes.
Better answer: As far as I'm aware, `type.__dict__.values()` is the only way of getting an iterator over all the properties of the class.

Reply:
Accessing magic isn't in itself a warcrime, either. Many of the functions which wrap it simply provide extra functionality, so they should be used when available.

Reply:
_grumbles_